internal/clients/kibana/slo_test.go

```diff
@@ -0,0 +1,33 @@
+package kibana
+
+import (
+	"testing"
+
+	"github.com/elastic/terraform-provider-elasticstack/generated/slo"
+	"github.com/elastic/terraform-provider-elasticstack/internal/models"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_sloResponseToModel(t *testing.T) {
+	tests := []struct {
+		name          string
+		spaceId       string
+		sloResponse   *slo.SloResponse
+		expectedModel *models.Slo
+	}{
+		{
+			name:          "nil response should return a nil model",
+			spaceId:       "space-id",
+			sloResponse:   nil,
+			expectedModel: nil,
+		},
+	}
```

Comment:
Shall we add a test case with data in `sloResponse` and `expectedModel`?

Reply:
Good idea, I had something there before, but didn't add it back for some reason.